tsdb/db_test.go

```diff
@@ -1111,11 +1113,58 @@ func TestSizeRetention(t *testing.T) {
 		createBlock(t, db.Dir(), genSeries(100, 10, m.MinTime, m.MaxTime))
 	}
 
+	headBlocks := []*BlockMeta{
+		{MinTime: 700, MaxTime: 800},
+		{MinTime: 800, MaxTime: 900},
```

Comment:
this doesn't create 2 wal files.

you can use with the WALSegmentSize to ensure that appending created more than a single file and it will best to add to the test that after appending to the wal it results in 2 files on disk.

Reply:
Any particular reason you think that having more WAL files will help the test? I don't expect the behaviour of the WAL size calculations will change as such, considering when creating the checkpoint it takes into account multiple files and dirs

Reply:
oh, I thought you have headBlocks multiple heads because you want to test the checkpointing with multiple WAL files

So why do you have 2 headBlocks than?

Reply:
No particular reason, although I do see now how that could cause some confusion. Do you think I should bring it down to just 1 head block?

Reply:
yep, and probbaly no need to use addfast for such small amount of samples.